### PR TITLE
[ROCKETMQ-207]Correct spelling mistake in RouteInfoManager.java

### PR DESCRIPTION
**Pull request by aCoder2013**
There is several spelling mistakes, it's a little confusing and i think we should fix them,like this one :
```java
if (null == prevBrokerLiveInfo) {
                   log.info("new broker registerd, {} HAServer: {}", brokerAddr, haServerAddr);
}
```
and here is the JIRA link of this issue : https://issues.apache.org/jira/browse/ROCKETMQ-207 ,thanks for your time

### CODE REVIEW

**Comment by Jaskey** [comment]
Thank you! Please relate JIRA  link in the discriptions.

Also, there are multiple typo string literal in rocektmq code base , it's welcome to fix them in this pr too.

**Comment by dongeforever** [comment]
LGTM @zhouxinyu @vongosling please help a review

File: namesrv/src/main/java/org/apache/rocketmq/namesrv/routeinfo/RouteInfoManager.java [hunk 1]
```diff
@@ -205,7 +205,7 @@ private void createAndUpdateQueueData(final String brokerName, final TopicConfig
             queueDataList = new LinkedList<QueueData>();
             queueDataList.add(queueData);
             this.topicQueueTable.put(topicConfig.getTopicName(), queueDataList);
-            log.info("new topic registerd, {} {}", topicConfig.getTopicName(), queueData);
+            log.info("new topic registered, {} {}", topicConfig.getTopicName(), queueData);
```
[the remark this answers is not in the document]
Comment (aCoder2013) [reply namesrv/src/main/java/org/apache/rocketmq/namesrv/routeinfo/RouteInfoManager.java]: @Jaskey Fixed ,man

**Comment by aCoder2013** [comment]
@zhouxinyu I've fixed the code style , please check it out ,thanks

**Pull request merged**